e2e/utils_test.go

```diff
@@ -41,7 +42,7 @@ func (g *gdProcess) Stop() error {
 	return g.Cmd.Process.Kill()
 }
 
-func (g *gdProcess) UpdateDirs() {
+func (g *gdProcess) updateDirs() {
```

Comment:
In my perfect world unexporting UpdateDirs would have been a separate commit, but that's mostly just a nitpick.